File: src/main/java/com/fireblazerrr/survivorbot/chatter/ChatterStorage.java
package com.fireblazerrr.survivorbot.chatter;

import java.util.UUID;

public interface ChatterStorage {

    void flagUpdate(Chatter chatter);

    Chatter load(UUID userUID);

    void removeChatter(Chatter chatter);

    void update();

    void update(Chatter chatter);

}

File: src/main/kotlin/com/fireblazerrr/survivorbot/chatter/ChatterStorage.kt
package com.fireblazerrr.survivorbot.chatter

import java.util.*

interface ChatterStorage{

    fun flagUpdate(chatter: Chatter): Nothing
    fun load(uuid: UUID): Chatter
    fun removeChatter(chatter: Chatter): Nothing
    fun update(): Nothing
    fun update(chatter: Chatter): Nothing
}